### PR TITLE
Kongtan operation panel

### PR DESCRIPTION
**Pull request by kouyjes**
### 👀 PR includes

✨ Feature

- [x] update OperatorBar 

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description
OperatorBar，support for multiple groups

### 🖼️ Screenshot
image

| Before | After |
| ------ | ----- |
| ❌     | ✅    |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/gi-assets-basic/src/components/OperatorBar/Component.tsx [hunk 1]
```diff
@@ -1,55 +1,213 @@
...
+    style.height = size;
+    style.left = left || 0;
+    style.right = right || 0;
+    if(position === 'Top'){
+      style.top = 0;
+    }else{
+      style.bottom = 0;
+    }
+  }
+  return ReactDOM.createPortal(<div className={className} style={{...style,...propStyle}}>
```
Comment (pomelo-nwu) [inline packages/gi-assets-basic/src/components/OperatorBar/Component.tsx:110]: ReactDOM.createPortal 的方式逐渐要废弃掉了，它主要的问题在于，当我们需要页面整体布局的时候，只能通过操作JSDOM来实现，造成开发成本提高也不利于维护，后续我们要逐渐去掉。新的方案可以一起review下这个：
https://github.com/antvis/G6VP/blob/v2/packages/gi-assets-basic/src/components/UadLayout/component.tsx#L28
https://github.com/antvis/G6VP/blob/v2/packages/gi-assets-basic/src/components/UadLayout/info.ts#L8
https://github.com/antvis/G6VP/blob/v2/packages/gi-sdk/src/GISDK.tsx#L366

File: packages/gi-assets-basic/src/components/OperatorBar/registerMeta.ts [hunk 2]
```diff
@@ -1,30 +1,284 @@
...
+      title: 'Left',
+      type: 'string',
+      'x-decorator': 'FormItem',
+      'x-component': 'Input',
+      'x-reactions': {
+        dependencies: ['.position'],
+        fulfill: {
+          schema: {
+            "x-visible": "{{$deps[0] === 'Top' || $deps[0] === 'Bottom'}}"
+          }
```
Comment (pomelo-nwu) [inline packages/gi-assets-basic/src/components/OperatorBar/registerMeta.ts:138]: 太强了，bty，Formily 这个表单写起来确实太费劲了...

**Review by pomelo-nwu — APPROVED** [review]

**Comment by pomelo-nwu** [comment]
@kouyjes 只需要把资产改个名字，这边就可以合并进来，同时基于新的GICC_LAYOUT类型，可以整体重构我们的一些容器资产

File: packages/gi-assets-basic/src/components/OperatorBar/Component.tsx (hunk 1 above)
Comment (kouyjes) [reply packages/gi-assets-basic/src/components/OperatorBar/Component.tsx:110]: 好，到时候，把相关的容器再适配一下

**Pull request merged**